test/WebSites/CustomUIConfig/wwwroot/ext/custom-plugin.js

```diff
@@ -6,6 +6,5 @@ const customPlugin = function (system) {
 
     document.body.insertBefore(elem, document.body.firstChild);
     return {
-         
     }
```

Comment:
```suggestion
    };
```

Reply:
![image](https://github.com/user-attachments/assets/3087aed9-eb32-4459-93ec-1b63979c6362)

Reply:
That's an entirely different line of code.